Ginger/GingerCoreCommon/Run/RunSetConfig.cs

```diff
@@ -86,6 +86,148 @@ public string Description
             }
         }
 
+        //Gideon
+        private bool mDefaultTestStageRadioBtn;
+        [IsSerializedForLocalRepository]
+        public bool DefaultTestStageYN
+        {
+            get { return mDefaultTestStageRadioBtn; }
+            set
+            {
+                if (mDefaultTestStageRadioBtn != value)
+                {
+                    mDefaultTestStageRadioBtn = value;
+                    OnPropertyChanged(nameof(DefaultTestStageYN));
+                }
+            }
+        }
+
+        private bool mCustomTestStageRadioBtn;
+        [IsSerializedForLocalRepository]
+        public bool CustomTestStageYN
+        {
+            get { return mCustomTestStageRadioBtn; }
+            set
+            {
+                if (mCustomTestStageRadioBtn != value)
+                {
+                    mCustomTestStageRadioBtn = value;
+                    OnPropertyChanged(nameof(CustomTestStageYN));
+                }
+            }
+        }
+
+        private bool mDefaultLabIdRadioBtn;
+        [IsSerializedForLocalRepository]
+        public bool DefaultLabIdYN
+        {
+            get { return mDefaultLabIdRadioBtn; }
+            set
+            {
+                if (mDefaultLabIdRadioBtn != value)
+                {
+                    mDefaultLabIdRadioBtn = value;
+                    OnPropertyChanged(nameof(DefaultLabIdYN));
+                }
+            }
+        }
+
+        private bool mCustomLabIdRadioBtn;
```

Comment:
1- remove all NY properties, keep only the customer ones
2- make sure name of property and matching local field starting with m having the dame name
3- make the customer NULL in case user shifted to default